source.ipynb

```diff
@@ -64,12 +74,18 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 1,
+   "attachments": {},
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Start your code here"
+    "I could not get my imports to work, however I would import pandas and beautiful soup to retreieve my data.\n",
+    "\n",
+    "I would download the openpowerlifting.csv file from Kaggle using opendatasets\n",
+    "\n",
+    "To open the text file I would use the with open function in read mode.\n",
```

Comment:
I would actually recommend you use one of the pandas built-in functions. 

the open function would require a lot more code.